src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -18,37 +20,83 @@
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the assignment identified by the index number used in the displayed assignment list.\n"
-            + "Parameters: INDEX (must be a positive integer and not greater than the size of the current "
+            + ": Deletes the assignment identified by the index number(s) used in the displayed assignment list.\n"
+            + "Parameters: INDEX [MORE INDEXES] (must be a positive integer, "
+            + "must not contain duplicates and cannot be greater than the size of the current "
             + "assignment list)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Example: " + COMMAND_WORD + " 1 2";
 
-    public static final String MESSAGE_DELETE_TASK_SUCCESS = "Deleted Assignment: %1$s";
+    public static final String MESSAGE_DELETE_TASK_SUCCESS = "Deleted Assignment(s): %1$s";
+    public static final String MESSAGE_ASSIGNMENTS_DUPLICATED_INDEX = "Duplicated indexes found.";
 
-    private final Index targetIndex;
+    private static final Comparator<Index> compareIndexes = (firstIndex, secondIndex) -> {
```

Comment:
I think should follow the naming convention and call this indexComparator, but just minor nitpicking

Reply:
Amended!